.github/workflows/subctl-unit.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Downstream Compatibility Check
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v**'
+
+jobs:
+  unit-testing:
+    name: subctl Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the submariner-operator repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          path: submariner-operator
+
+      - name: Check out the subctl repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          repository: submariner-io/subctl
+          path: subctl
+
+      - name: Check out the shipyard repository
```

Comment:
Why is Shipyard needed for this one?

Reply:
We need to run a “local build” because we need several repositories (since only the repository running `.dapper` is made available inside the container), which means we need a local clone of the Shipyard repository. I’ll add a comment explaining this.